test/test_build.py

```diff
@@ -452,6 +453,25 @@ def test_iso_installs(image_type):
         assert_kernel_args(vm, image_type)
 
 
+@pytest.mark.skipif(platform.system() != "Linux", reason="osinfo detect test only runs on linux right now")
+@pytest.mark.parametrize("image_type", gen_testcases("anaconda-iso"), indirect=["image_type"])
+def test_iso_os_detection(image_type):
+    installer_iso_path = image_type.img_path
+    arch = image_type.img_arch
+    template = image_type.osinfo_template
+    if not arch:
```

Comment:
(nitpick) I would move this line right next to "arch = image_type.img_arch" as they belong together